docs/run_test/platform.md

```diff
@@ -0,0 +1,163 @@
+# Run LISA
```

Comment:
If a commit is used to copy file, it needs to write in commit description. It can help others know where the content is from, and skip to review it.